repose-aggregator/components/services/opentracing-service/opentracing-service-impl/out/production/resources/META-INF/schema/config/opentracing.xsd

```diff
@@ -0,0 +1,135 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
As is this one. Besides being in the wrong directory, they are on the wrong project, they should be in the api.

Reply:
I honestly had no idea where this needed to live.  moving!